apps/wpcom-block-editor/src/wpcom/features/live-preview/hooks/use-can-preview-but-need-upgrade.ts

```diff
@@ -139,10 +138,7 @@ export const useCanPreviewButNeedUpgrade = (
 
 	const handleCanPreviewButNeedUpgrade = useCallback(
 		( previewingTheme: ReturnType< typeof usePreviewingTheme > ) => {
-			const livePreviewUpgradeTypes = [ WOOCOMMERCE_THEME, PREMIUM_THEME ];
-			if ( config.isEnabled( 'themes/tiers' ) ) {
-				livePreviewUpgradeTypes.push( PERSONAL_THEME );
-			}
+			const livePreviewUpgradeTypes = [ WOOCOMMERCE_THEME, PREMIUM_THEME, PERSONAL_THEME ];
```

Comment:
Since we can't have feature flags in the block editor we need to always add the tier.